specs/lightclient/lightclient-fork.md

```diff
@@ -66,10 +67,8 @@ def upgrade_to_lightclient_patch(pre: phase0.BeaconState) -> BeaconState:
         # Slashings
         slashings=pre.slashings,
         # Attestations
-        # previous_epoch_attestations is cleared on upgrade. 
-        previous_epoch_attestations=List[PendingAttestation, MAX_ATTESTATIONS * SLOTS_PER_EPOCH](),
-        # empty in pre state, since the upgrade is performed just after an epoch boundary.
-        current_epoch_attestations=List[PendingAttestation, MAX_ATTESTATIONS * SLOTS_PER_EPOCH](),
+        previous_epoch_participation=[Bitvector[PARTICIPATION_FLAGS_LENGTH]() for _ in range(len(pre.validators))],
```

Comment:
Was it intentional to drop the history of attestations from the previous epoch?

Reply:
The alternative is to write a function to convert pending attestations into flags. This has relatively high consensus complexity for a single transition epoch. 

One alternative is to just give all validators +base_reward at the epoch transition to avoid the complexity of the logic here...